### PR TITLE
scheme.go: Skip debug directory

### PR DESCRIPTION
**Pull request by metalmatze**
This is a rebased version of https://github.com/observatorium/thanos-replicate/pull/14.

### CODE REVIEW

**Comment by metalmatze** [comment]
PTAL

File: scheme.go [hunk 1]
```diff
@@ -160,6 +160,12 @@ func (rs *replicationScheme) execute(ctx context.Context) error {
 	err := rs.fromBkt.Iter(ctx, "", func(name string) error {
 		rs.metrics.originIterations.Inc()
 
+		// Filter debug block
+		_, ok := thanosblock.IsBlockDir(name)
```
Comment (bwplotka) [inline scheme.go]: but id is given here so why we do this `ulidString := name[:len(name)-1]` `id, err := ulid.Parse(ulidString)` steps?

Comment (brancz) [reply scheme.go]: done in #16, sorry for the chaos :upside_down_face:

**Comment by brancz** [comment]
Sorry for the chaos, continued in #16

**Pull request closed without merging**